tests/e2e_regression/run.sh

```diff
@@ -36,6 +36,9 @@ testCases=(
   'account                  /v1/consensus/accounts/oasis1qp0302fv0gz858azasg663ax2epakk5fcssgza7j'
   'runtime-only_account     /v1/consensus/accounts/oasis1qphyxz5csvprhnn09r49nuyzl0jdw0wsj5xpvsg2'
   'delegations              /v1/consensus/accounts/oasis1qp0302fv0gz858azasg663ax2epakk5fcssgza7j/delegations'
+  'delegations_to           /v1/consensus/accounts/oasis1qp0j5v5mkxk3eg4kxfdsk8tj6p22g4685qk76fw6/delegations_to'
```

Comment:
Only these three lines were updated.

Other changes are autofixes done by my editor (_i think_ that by `prettier`, can revert if desired).

Reply:
:shrug: I think line 109 was clearer pre-`prettier` because it visually separated `cp` and `jq`, but I don't care enough. SGTM to leave as-is.

Thanks for adding the tests!